aait/backend/group-2B/Application/DTOs/Users/UserDeletDto.cs

```diff
@@ -0,0 +1,11 @@
+
+using SocialSync.Application.DTOs.Common;
+
+namespace Application.DTOs.Users
+{
+    public class UserDeletDto : BaseDto
+    {
+        
+        public int Owner { get; set; }
+    }
+}
```

Comment:
Formatting should be improved. We are using
```csharp
namespace Some.Name.Space;

class SomeClass
{
    ...
}
```

rather than:
```csharp
namespace Some.Name.Space
{
    class SomeClass
    {
        ....
    }
}
```

You can achieve all of that for all of your files using `Ctrl+Shift+P` -> Search `Format Document` -> Hit Enter on VS Code.